Polygon_mesh_processing/doc/Polygon_mesh_processing/Concepts/PMPAutorefinementVisitor.h

```diff
@@ -0,0 +1,25 @@
+/// \ingroup PkgPolygonMeshProcessingConcepts
+/// \cgalConcept
+///
+/// The concept `PMPAutorefinementVisitor` defines the requirements for the visitor
+/// used in `CGAL::Polygon_mesh_processing::autorefine_triangle_soup()` to track
+/// the creation of new triangles.
+///
+/// \cgalRefines{CopyConstructible}
+/// \cgalHasModel `CGAL::Polygon_mesh_processing::Autorefinement::Default_visitor`.
```

Comment:
In combination with #7395 this gives at the moment a warning
```
/home/cgal-testsuite/cgal_doc_build/CGAL-6.0-Ic-51/doc/Polygon_mesh_processing/Concepts/PMPAutorefinementVisitor.h:9: warning: Found unknown command '\cgalHasModel'
```
as the structure for `\cgalHasModel` has changed and this here is not adapted to this new structure (it is a new file).